docs/ops/comparison/IsFinite_10.md

```diff
@@ -0,0 +1,45 @@
+# IsFinite {#openvino_docs_ops_comparison_IsFinite_10}
+
+**Versioned name**: *IsFinite-10*
+
+**Category**: *Comparison*
+
+**Short description**: *IsFinite* performs element-wise test for finiteness (not Infinity and not NaN) on elements of a given input tensor. It returns the boolean mask of a given tensor which maps
+`NaN` and `Infinity` to `False` and all other values to `True`
+*IsFinite* operation has no attributes.
+
+**Detailed description**: *IsFinite* returns the bolean mask of the input tensor in which `False` corresponds to `NaN` and `Infinity` and `True` to all other values.
+* The output tensor has the same shape as input tensor.
+* The `i`'th element of the output tensor is `False` if `i`'th element of the input tensor is `NaN` or `Inf`. Otherwise it is `True`.
+* For example, for given input tensor `[NaN, 2.1, 3.7, Inf]` the output tensor is `[False, True, True, False]`.
```

Comment:
(Optional) Example values can be also added to the "Examples" section.
Reference: [Select_1.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/condition/Select_1.md), [CumSum_3.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/CumSum_3.md)

Reply:
done